src/main/java/com/spartronics4915/frc2020/Constants.java

```diff
@@ -68,9 +72,9 @@
     public static final class Launcher
     {
         public static final int kFlywheelMasterId = 7;
-        public static final int kFlywheelFollowerId = -1; // Solid brass
-        public static final int kAngleAdjusterMasterId = 0; // PWM0
-        public static final int kAngleAdjusterFollowerId = 1; // PWM1
+        public static final int kFlywheelFollowerId = 0; // Solid brass
```

Comment:
Revert this to -1. 

-1 is the convention for invalid ids, which this is - it's only here as a placeholder if another motor does get added.